tests.py

```diff
@@ -14,6 +14,9 @@
 
 
 class TestVSM(unittest.TestCase):
+    def setUp(self):
+        print(self._testMethodName)
```

Comment:
Well it's good if you are outputting things to say what it was connected to. I'd say it's worth having.